client/scenes/lobby_gui.cpp

```diff
@@ -757,16 +758,29 @@ void scenes::lobby::gui_settings()
 			}
 			else
 			{
-				if (application::get_fb_body_tracking_supported())
-					tooltip(_("Requires 'Hand and body tracking' to be enabled in the Quest movement tracking settings,\notherwise body data will be guessed from controller and headset positions"));
-				else if (application::get_pico_body_tracking_supported())
-					tooltip(_("Uses the Pico Motion Trackers to track body joint positions"));
+				std::string tooltip_msg = std::visit(utils::overloaded{
+				                                             [&](std::monostate &) {
+					                                             return std::string{};
+				                                             },
+				                                             [&](xr::fb_body_tracker &) {
+					                                             return _("Requires 'Hand and body tracking' to be enabled in the Quest movement tracking settings,\notherwise body data will be guessed from controller and headset positions");
+				                                             },
+				                                             [&](xr::htc_body_tracker &) {
+					                                             return _("Forwards positions of Vive Ultimate Trackers");
+				                                             },
+				                                             [&](xr::pico_body_tracker &) {
+					                                             return _("Uses the Pico Motion Trackers to track body joint positions");
+				                                             },
+				                                     },
+				                                     application::get_body_tracker());
```

Comment:
Does the lambda need to capture anything? the `&` can be removed.

Or just call `tooltip` inside the lambda, and don't return a value.